x11-wm/pekwm/pekwm-0.2.1.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib desktop optfeature
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="debug +jpeg +png truetype +xft xinerama +xpm"
```

Comment:
`debug` and `xft` are not being used in this ebuild. Please re-check especially the `xft` flag.

Reply:
Debug is supported by pekwm, so I add this via CMAKE_BUILD_TYPE=$(usex debug Debug). xft removed. Commit b75cdc3.